src/aleph/web/controllers/ipfs.py

```diff
@@ -29,7 +30,23 @@ async def ipfs_add_file(request: web.Request):
     except KeyError:
         raise web.HTTPUnprocessableEntity(reason="Missing 'file' in multipart form.")
 
-    ipfs_add_response = await ipfs_service.add_file(file_field_to_io(file_field))
+    file_content: bytes
+    if isinstance(file_field, bytes):
+        file_content = file_field
+    elif isinstance(file_field, str):
+        file_content = file_field.encode()
```

Comment:
Can we for sure expect the file content to be utf-8 encoded? What about base64? What's the convention behind this?